src/PhpParser/NodeVisitor/NameStmtPrefixer.php

```diff
@@ -204,4 +226,14 @@ private function prefixName(Name $name): Node
             $resolvedName->getAttributes()
         );
     }
+
+    private static function array_starts_with($arr, $prefix): bool
+    {
+        for ($i = 0; $i < count($prefix); ++$i) {
```

Comment:
be aware that the count here is executed every single time which is not ideal

Reply:
Ha, forgot I was in one of the languages that does this, thanks! :)